front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -155,6 +155,15 @@ export default function AddOrEditStudyModal({ editionMode, study }: Props) {
     }
   }, [isCreateStudyError, isPatchStudyError, isDeleteStudyError]);
 
+  const isInvalidDate = () => {
```

Comment:
Use a useMemo instead of a function
```suggestion
  const isInvalidDate = useMemo(() => {...}, [currentStudy.start_date, currentStudy.end_date]);
```

Also, I think it would be nice to have 3 const (use useMemo for the 3):
- isExpectedEndDateValid
- isActualEndDateValid
- areEndDatesValid = isExpectedEndDateValid && isActualEndDateValid

Reply:
Fixed ! Ty for the suggestions :)